packages/taiko-client/driver/chain_syncer/blob/soft_block.go

```diff
@@ -248,16 +250,31 @@ func (s *Syncer) InsertSoftBlockFromTransactionsBatch(
 		return nil, err
 	}
 
-	log.Info(
-		"⏰ New soft L2 block inserted",
-		"blockID", blockID,
-		"hash", header.Hash(),
-		"transactions", len(payload.Transactions),
-		"baseFee", utils.WeiToGWei(header.BaseFee),
-		"withdrawals", len(payload.Withdrawals),
-		"endOfBlock", attributes.L1Origin.EndOfBlock,
-		"endOfPreconf", attributes.L1Origin.EndOfPreconf,
-	)
+	if batchID == 0 {
+		log.Info(
+			"⏰ New soft L2 block inserted",
+			"batchID", batchID,
```

Comment:
don't think this is necessary, we can just use `⏰ New soft L2 block inserted` and check the `batchID`